lib/kubeclient/kube_exception.rb

```diff
@@ -9,6 +9,10 @@ def initialize(error_code, message, response)
   end
 
   def to_s
-    'HTTP status code ' + @error_code.to_s + ', ' + @message
+    string = "HTTP status code #{@error_code}, #{@message}"
+    if @response.is_a?(RestClient::Response) && @response.request
+      string << " for #{@response.request.method.upcase} #{@response.request.url}"
```

Comment:
LGTM.  RestClient::Request [normalizes method][1] to always be a string.

[1]: https://github.com/rest-client/rest-client/blob/f62a8a0e/lib/restclient/request.rb#L707-L710